PS3GUD.py

```diff
@@ -31,6 +31,8 @@
 import requests.adapters
 import keyring
 
+requests.packages.urllib3.disable_warnings(category=InsecureRequestWarning)
```

Comment:
please remove this
atleast the user should be informed that bypassing ssl checks are insecure